### DISCUSSION TITLE
Add man page entries

### DISCUSSION THREAD

**Discussion by Nikaidou-Shinku** [xmake-io/xmake#3170] [Ideas]
I first followed this project because it quickly provided support for C++20 modules, and I've been receiving information about xmake ever since. Anyway, thanks a lot to you for maintaining such a great tool!

I recently started trying to build my new project using xmake, but I found that it doesn't provide man pages. For users of Unix-like operating systems, access to man pages is a common need. Do you have any further considerations for this? Or maybe the community can provide man pages for xmake?

**Comment by waruqi** [comment]
I don't have the extra time to add and maintain manual pages, but I accept and welcome contributors to submit pr for it.

**Comment by Nikaidou-Shinku** [comment]
Well, maybe other people in the xmake community can contribute, thank you anyway!
